17thblog/loggedIn.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+include('config/config.php');
+include('config/init.php');
+
+echoHeader("Logged In", "Logged In");
+
+echo "<div class='textStyle form'>";
+
+//make a functino called verify user as lgoged in. so any page, u can call that function at top
+if(!isset($_SESSION['Username'])){
```

Comment:
At some point you'll want to make this into a function called `verifyUserIsLoggedIn` or something like that so you can call it at the top of every page that requires a user to be logged in. You also might want to automatically redirect the user to a login page.